.travis.yml

```diff
@@ -49,8 +51,8 @@ script:
       make get-deps;
     fi;
     if [ "$TRAVIS_GO_VERSION" == "tip" ] ||
-        [ "$TRAVIS_GO_VERSION" == "1.13.x" ] ||
-        [ "$TRAVIS_GO_VERSION" == "1.12.x" ]; then
+        [ "$TRAVIS_GO_VERSION" == "1.14.x" ] ||
+        [ "$TRAVIS_GO_VERSION" == "1.13.x" ]; then
```

Comment:
Should add version 1.12.x

Reply:
Not needed in this case, because the `unit-old-go-race-cover` step doesn't run the verify recipe.